### ISSUE TITLE
Sandstorms have a huge count of tumbleweeds/branches and are laggy

### ISSUE THREAD

**Issue by Gliese-832-c** [Mrbt0907/Weather2-Remastered#135]
For some reason sandstorms have a large amount of tumbleweeds and branches. On top of looking unrealistic and ugly, this causes a large FPS drop.
![image](https://user-images.githubusercontent.com/55159077/227632144-b784cf3f-69de-4f11-9976-f54a47856193.png)
I suggest a severe reduction of the amount of tumbleweeds and branches, probably to 1% or less of the current amount.

**Comment by CosmicHikari** [comment]
Yes, sandstorms are currently still a work in progress. They should be changed by the end of 2.8 development.

**Comment by Mrbt0907** [comment]
This should be fixed by the next alpha release. This happened because I changed the particle system a lot, making out of date storms like sandstorms really messy.
